### ISSUE TITLE
Full-screen screenshots broken

### ISSUE THREAD

**Issue by SparkyBluefang** [linuxmint/cinnamon#10147]
```
Cinnamon 5.0.2
Gentoo
Quadro RTX 3000 nvidia-drivers 460.67
64 bit
 ```

**Issue**

Full-screen screenshot only contains the upper-left 1/4 of the screen. The rest of it is black.

The only log messages I see look to be the normal X11 fallback noise.
```
** Message: 18:18:16.372: Unable to use GNOME Shell's builtin screenshot interface, resorting to fallback X11.
```

![Screenshot from 2021-06-08 18-10-23](https://user-images.githubusercontent.com/3150882/121265106-40025680-c886-11eb-8c9f-1b8d255c5ffe.png)

**Comment by icarter09** [comment]
What tool were you using to take the screenshots?

**Comment by SparkyBluefang** [comment]
gnome-screenshot 40.0

I get this behavior both with Cinnamon's PrintScr handling and by launching `gnome-screenshot -i` and selecting the "screen" capture option. The "window" (or Alt+PrintScr) and "selection" options don't seem to have problems.

**Comment by SparkyBluefang** [comment]
Oh, since I've forgotten both times I've posted - this is a HiDIP screen. 3840x2160 with a 2x scale.

And, of course, after a bit of searching, I did find this:
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/128

Interestingly, this problem should affect both 40.0 and 38.0, which is the version I was using up until recently. I don't remember this being a problem with 38.0, but I honestly don't know if I've taken a screenshot since upgrading from 36.0...

So I've got some version testing to do to see if this is actually a Cinnamon issue, or if gnome-screenshot was just broken all along X_X
